#!/usr/bin/env python
PACKAGE = "phobos_control_manipulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("uzyj_pozycji", bool_t, 0, "description",    False)
gen.add("stop", bool_t, 0, "description",    False)

gen.add("ustaw_zero_link0", bool_t, 0, "description",    False)
gen.add("ustaw_zero_link1", bool_t, 0, "description",    False)
gen.add("ustaw_zero_link2", bool_t, 0, "description",    False)
gen.add("ustaw_zero_link3", bool_t, 0, "description",    False)
gen.add("ustaw_zero_link4", bool_t, 0, "description",    False)
gen.add("ustaw_zero_grip", bool_t, 0, "description",    False)


gen.add("jedz_w_lewo_link0", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_link0", bool_t, 0, "description",    False)

gen.add("jedz_w_lewo_link1", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_link1", bool_t, 0, "description",    False)

gen.add("jedz_w_lewo_link2", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_link2", bool_t, 0, "description",    False)

gen.add("jedz_w_lewo_link3", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_link3", bool_t, 0, "description",    False)

gen.add("jedz_w_lewo_link4", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_link4", bool_t, 0, "description",    False)

gen.add("jedz_w_lewo_grip", bool_t, 0, "description",    False)
gen.add("jedz_w_prawo_grip", bool_t, 0, "description",    False)

gen.add("otworz_pojemnik_L", bool_t, 0, "description",    False)
gen.add("zamknij_pojemnik_L", bool_t, 0, "description",    False)

gen.add("otworz_pojemnik_S", bool_t, 0, "description",    False)
gen.add("zamknij_pojemnik_S", bool_t, 0, "description",    False)

gen.add("otworz_pojemnik_P", bool_t, 0, "description",    False)
gen.add("zamknij_pojemnik_P", bool_t, 0, "description",    False)


exit(gen.generate(PACKAGE, "phobos_control_manipulator", "ControlMsgs"))
